exercises/numbers/isOdd/isOdd.js

```diff
@@ -14,10 +14,16 @@
  * @returns {boolean} True if num is even and false otherwise
  */
 function isOdd(num) {
-  // Hint: See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder
-  return _____;
+  if (num % 2 === 0) {
```

Comment:
Consider using `return num % 2 == 1 ? true : false`, one line solution! `?` is called **ternary operator** and it simplifies if conditions. It's used like this: **condition ? exprIfTrue : exprIfFalse**